gallery/src/components/NxDrawer/NxDrawerEscExample.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+
+import { NxDropdown,
+  NxDrawer,
+  NxButton,
+  useToggle,
+  NxBackButton,
+  NxPageTitle,
+  NxH1,
+  NxFontAwesomeIcon,
+  NxPageMain,
+  NxP
+
+} from '@sonatype/react-shared-components';
+
+import { faEdit } from '@fortawesome/free-solid-svg-icons';
+
+export default function NxDrawerEscExample() {
+  const [showDrawer, setShowDrawer] = useState(false);
+  const [dropdownIsOpen, onToggleDropdownCollapse] = useToggle(false);
+
+  const onClick = () => { alert('click'); };
+
+  return (
+    <>
+      <header className="nx-global-header">
+        <NxBackButton href="#/pages/Drawer" targetPageTitle="Documentation" />
+        <div className="nx-global-header__actions">
+          <NxButton title="Edit" variant="icon-only"><NxFontAwesomeIcon icon={faEdit} /></NxButton>
+        </div>
+      </header>
+      {showDrawer && (
+        <NxDrawer id="nx-drawer-esc"
+                  onCancel={() => setShowDrawer(false)}
+                  aria-label="Drawer with Escape Example">
+          <NxDrawer.Header>
+            <NxDrawer.Header.Title>Header Title</NxDrawer.Header.Title>
+            <NxDrawer.Header.Subtitle>Header Subtitle</NxDrawer.Header.Subtitle>
+            <NxDrawer.Header.Description>Header Description</NxDrawer.Header.Description>
```

Comment:
Doubly nesting component names hasn't been our convention so far.  For the sake of consistency, let's make these `NxDrawer.HeaderTitle` etc.

Reply:
> Doubly nesting component names hasn't been our convention so far. For the sake of consistency, let's make these `NxDrawer.HeaderTitle` etc.

Updated tests here: c76e0100063868e482e53900c157b5eab5712a88